tests/test_browser.py

```diff
@@ -4338,7 +4338,9 @@ def test_base64_atob_fallback(self):
   # Tests that SINGLE_FILE works as intended in generated HTML (with and without Worker)
   def test_single_file_html(self):
     self.btest('emscripten_main_loop_setimmediate.cpp', '1', args=['-s', 'SINGLE_FILE=1', '-s', 'WASM=1'], also_proxied=True)
-    assert os.path.exists('test.html') and not os.path.exists('test.js') and not os.path.exists('test.worker.js')
+    self.assertExists('test.html')
```

Comment:
That should make it easier to tell what went wrong than having to look at all the `and`ed clauses